### PR TITLE
Add support for data sets

### PR DESCRIPTION
**Pull request by emelvarol-cognite**
We only have the option for setting/updating data sets currently. I noticed the data_set_id.remove is not implemented in the SDK. I'll fix that and use it in the CLI.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -12,6 +12,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [1.3.2] - 2022-02-14
+### Added
+- Support setting data set ids on transformations.
```
Comment (hakontro) [inline CHANGELOG:17]: Is data set one or two words? :thinking: I don't know, but mostly see it as `datasets` I think?

**Review by hakontro — APPROVED** [review]
Yes

File: CHANGELOG (hunk 1 above)
Comment (emelvarol-cognite) [reply CHANGELOG:17]: It changes a lot, i still don't know. We used it as `DataSets` in jetfire sometimes, and "datasets" in some places as it seems 🤯  
In my thesis, overleaf has put red lines when I wrote datasets as well 🤣  What should we do??????

Comment (emelvarol-cognite) [reply CHANGELOG:17]: image

Comment (vuhainguyen) [reply CHANGELOG:17]: No need to fight both are correct according to [wikipedia](https://en.wikipedia.org/wiki/Data_set) =))

**Pull request merged**